src/app/components/layout/header/top-bar/top-bar.component.scss

```diff
@@ -9,7 +9,7 @@
 .buttons-left {
   display: inline-block;
   padding: 0 10px;
-  width: 200px;
+  width: 220px;
```

Comment:
If you change this, you also have to change the width of `.buttons-right`. Otherwise the title is stops being centered correctly.